CONTRIBUTING.md

```diff
@@ -58,6 +58,7 @@ We welcome icon requests. Before you submit a new issue please make sure the ico
     - Governmental agencies, programs, departments
        - Allowed: International organizations and NGOs with supranational interests
        - Allowed: Space agencies
```

Comment:
Trying to figure out how we can explicitly mention that "no military brands/orgs" overrules the allowed "international organizations and NGOs with supranational interests"...

Reply:
Note that "International organizations and NGOs with supranational interests" is mentioned in the context of governmental agencies. Organizations like the NATO seems to me that doesn't resides in that category.

Reply:
Totally agree, @mondeja though that does leave stuff like NATO in a grey area - and I think we're best to explicitly state we will not accept military-affiliated organisations.

Reply:
Yes, I agree. Just trying to state that doesn't overrules the previous "governmental" category, that could be a different category. I'm fine with the current wording but not being a native English speaker, let to you the approval.